readers/api/transport.go

```diff
@@ -39,16 +39,24 @@ const (
 	defLimit       = 10
 	defOffset      = 0
 	defFormat      = "messages"
+	thingToken     = "Thing "
```

Comment:
Why there is an empty space after the word? Does not seem quite right... Anyway, we should probably not use custom header tagsm as authentication schemes are limited to [these](https://developer.mozilla.org/en-US/docs/Web/HTTP/Authentication#authorization_and_proxy-authorization_headers).

Reply:
On the other hand - there are [variants](https://www.loginradius.com/blog/async/everything-you-want-to-know-about-authorization-headers/), and AWS is using [this](https://docs.aws.amazon.com/AmazonS3/latest/API/sigv4-auth-using-authorization-header.html) for example.

Reply:
@dusanb94 and I already discussed this and I think we agreed with custom header authentication as I saw that it is widely being used ( and looks like AWS example testifies that ).
My initial idea was just to use `Authorization: <token_value|thing_key>` or  `Authorization:Bearer <token_value|thing_key>`, if one fails try other. If discard that solution I don't know any other way than to use different prefix to specify different authentication.

Reply:
This is OK, but it should be `"Thing"`, not `"Thing "`.